Problems/1065/ans_1065_JHP.py

```diff
@@ -0,0 +1,26 @@
+# (Some test cases are failed. Need to update - 2021.11.24)
+# https://www.acmicpc.net/problem/1065
+
+# 문제 : 어떤 양의 정수 X의 각 자리가 등차수열이라면 그 수를 "한수"라함.
+# 등차수열은 연속된 두개의 수의 차이가 일정한 수열을 말함.
+# N이 주어졌을 때, 1보다 크거나 같고, N보다 작거나 같은 한수의 개수를 출력
+
+def solve():
+    num = int(input())
+    cnt = 0
+    
+    for val in range(1,num):
+        num_str = str(val)
+        if len(num_str) < 3:
+            cnt += 1
```

Comment:
```suggestion
        if val< 100
            cnt += 1
```
좀더 명확할듯?
혹은 100 이상의 input에서 100부터 cnt 시작이라든지..